strings/rand.go

```diff
@@ -3,18 +3,18 @@
 package strings
 
 import (
-	"strings"
-
 	"github.com/evenlab/go-kit/bytes"
+	"strings"
```

Comment:
plz make a fix by linter requirement.

Reply:
I have:
context/context.go:98:3: use of `print` forbidden by pattern `^(fmt\.Print(|f|ln)|print|println)$` (forbidigo)
what linter doesn't like about "print"

Reply:
This message contains the name of the linter - forbidigo. You could just ignore it for now.